cluster/gce/gci/configure-helper.sh

```diff
@@ -2713,13 +2713,18 @@ function main() {
 
   KUBE_HOME="/home/kubernetes"
   KUBE_BIN=${KUBE_HOME}/bin
-  PYTHON="python"
   CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
   PV_RECYCLER_OVERRIDE_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/pv-recycler-template.yaml"
 
-  if [[ "$(python -V)" =~ "Python 3" ]]; then
-    PYTHON="/usr/bin/python2.7"
```

Comment:
we don't fall back to python2.7 with the new approach, is that ok?

Reply:
Yes, this is ok. We want to identify CI jobs that have old python2.7 and need to switch them over